api/bro_upload/templates/registration_GMN_StartRegistration.html

```diff
@@ -0,0 +1,31 @@
+<registrationRequest xmlns="http://www.broservices.nl/xsd/isgmn/1.0"
+    xmlns:brocom="http://www.broservices.nl/xsd/brocommon/3.0"
+    xmlns:gml="http://www.opengis.net/gml/3.2"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.broservices.nl/xsd/isgmn/1.0 https://schema.broservices.nl/xsd/isgmn/1.0/isgmn-messages.xsd">
+    <brocom:requestReference>{{ metadata.requestReference }}</brocom:requestReference>
+    <brocom:deliveryAccountableParty>{{ metadata.deliveryAccountableParty }}</brocom:deliveryAccountableParty>
+    <brocom:qualityRegime>{{ metadata.qualityRegime }}</brocom:qualityRegime>
+    <sourceDocument>
+        <GMN_StartRegistration gml:id="id_0001">
+            <objectIdAccountableParty>{{ sourcedocs_data.objectIdAccountableParty }}</objectIdAccountableParty>
+            <name>{{ sourcedocs_data.name }}</name>
+            <deliveryContext codeSpace="urn:bro:gmn:DeliveryContext">{{ sourcedocs_data.deliveryContext }}</deliveryContext>
+            <monitoringPurpose codeSpace="urn:bro:gmn:MonitoringPurpose">{{ sourcedocs_data.monitoringPurpose }}</monitoringPurpose>
+            <groundwaterAspect codeSpace="urn:bro:gmn:GroundwaterAspect">{{ sourcedocs_data.groundwaterAspect }}</groundwaterAspect>
+            <startDateMonitoring>
+                <brocom:date>{{ sourcedocs_data.startDateMonitoring }}</brocom:date>
+            </startDateMonitoring>{% for measuringpoint in sourcedocs_data.measuringPoints %}
+            <measuringPoint>
```

Comment:
```suggestion
            </startDateMonitoring>
{%             for measuringpoint in sourcedocs_data.measuringPoints %}
            <measuringPoint>
```

Zo'n "for" is het duidelijkst op z'n eigen regel. 
Eventueel na de `%` spaties toevoegen om 'em duidelijker ergens bij te laten horen.